app/instance-initializers/violations-helper.js

```diff
@@ -0,0 +1,11 @@
+import ViolationsHelper from 'ember-a11y-testing/utils/violations-helper';
+
+export function initialize(application) {
+  window.violationsHelper = new ViolationsHelper();
```

Comment:
@Robdel12 That sounds good to me! Do you think this "feature" warrants a note in the README?